### ISSUE TITLE
Release Version with Neos 7 Support

### ISSUE THREAD

**Issue by paavo** [Sebobo/Shel.Crawler#2]
`dev-master` works well with Neos 5.3

I haven't tested it in Neos 7, but someone should ☝️

**Comment by paavo** [comment]
Oh, i didn't saw the 2.0.0 Version as it wasn't listed first 🤣 
2.0.0 works well with Neos 5.3

Returned here as im now im looking for Neos 7 support 🙈

**Comment by paavo** [comment]
I just tested `master` branch with Neos 7.3.0 and PHP 8.0. 
Everything looks good @Sebobo

**Comment by Sebobo** [comment]
I don't use it in that context yet.

Can you maybe debug which curl option is the problem?

**Comment by paavo** [comment]
Looks like $options is an empty array `array $options = []` @Sebobo 

https://github.com/Sebobo/Shel.Crawler/blob/389426404b00990ff96431b42f9340b9dbb5f12c/Classes/Service/SitemapService.php#L144

Dumping `$browser` shows some options 🤔 

```
Neos\Flow\Http\Client\Browser prototype object proxy
 lastRequest => NULL
 lastResponse => NULL
 followRedirects => true
 maximumRedirections => integer 10
 redirectionStack => array(empty)
 automaticRequestHeaders => Neos\Flow\Http\Headers
  string "Cache-Control" (13) => array(empty)
 requestEngine => Neos\Flow\Http\Client\CurlEngine prototype object proxy
  options => array(11)
    integer 19913 => true
    integer 42 => true
    integer 84 => integer 2
    integer 74 => true
    integer 75 => true
    integer 13 => integer 30
    integer 0 => integer 1
    integer 1 => integer 30
    integer 2 => integer 1
    integer 3 => integer 0
    integer 4 => integer 0
 serverRequestFactory => Neos\Http\Factories\ServerRequestFactory prototype object proxy
  uriFactory => Neos\Http\Factories\UriFactory prototype object proxy
  defaultUserAgent => string "Flow/7.3" (8)
  scriptPath => string "index.php" (9)
  defaultHttpVersion => string "1.1" (3)
 contentStreamFactory => Neos\Http\Factories\StreamFactory prototype object proxy
 Flow_Injected_Properties => array(2)
   integer 0 => string "serverRequestFactory" (20)
   integer 1 => string "contentStreamFactory" (20)

```

**Comment by Sebobo** [comment]
Thx, fixed in 2.2.1

**Comment by paavo** [comment]
thank you very much @Sebobo 👍 🎉
